src/hotspot/share/prims/jvmti.xml

```diff
@@ -8249,44 +8249,53 @@ class C2 extends C1 implements I2 {
         <field id="owner">
           <jthread/>
             <description>
-              The thread owning this monitor, or <code>nullptr</code> if unused
+              The platform thread owning this monitor, or <code>nullptr</code> if owned
+              by a virtual thread or not owned
             </description>
         </field>
         <field id="entry_count">
           <jint/>
           <description>
-            The number of times the owning thread has entered the monitor
+            The number of times the owning platform thread has entered the monitor
           </description>
         </field>
         <field id="waiter_count">
           <jint/>
           <description>
-            The number of threads waiting to own this monitor
+            The number of platform threads waiting to own this monitor,
+            or <code>0</code> if the monitor is owned by a virtual thread or not owned
```

Comment:
Good suggestion, thanks. But it is more "incorrect". It should say "is waited by" instead of "is owned by":
```
           The number of platform threads waiting to own this monitor, or <code>0</code>
           if the monitor is waited by virtual threads only or not owned
``` 
Are you okay with this correction?

Reply:
Copy and paste issue on my part. I would use "if only virtual threads".

Reply:
Okay, thanks.

Reply:
Second suggestion is better. "waited by" is not grammatically correct in this context.